MediaBrowser.Common/Providers/ProviderIdParsers.cs

```diff
@@ -18,7 +18,7 @@ public static class ProviderIdParsers
         /// <param name="text">The text to parse.</param>
         /// <param name="imdbId">The parsed IMDb id.</param>
         /// <returns>True if parsing was successful, false otherwise.</returns>
-        public static bool TryFindImdbId(ReadOnlySpan<char> text, [NotNullWhen(true)] out ReadOnlySpan<char> imdbId)
```

Comment:
Is this automatically applied now? Or why is it no longer required?

Reply:
`ReadOnlySpan<char>` can't be null, it's a (ref) struct